.example.env

```diff
@@ -0,0 +1,5 @@
+PORT={port number}
+
+JWT_SECRET_KEY={custom secret key for JWT sign}
+
+COINLAYER_API_KEY={api key for bitcoin rates api}
```

Comment:
EOF line is missing.

Reply:
I get it that EOF stands for End Of File. But I am not sure I get what exactly is expected. Could you please hint me with an example? :)

Reply:
It's usual for files with code to end with a blank line. I think it serves the same way as a comma in the vertical array/function arguments like this:
`const longArray = [
  '1',
  '2,235',
  '-1F',
];`

It makes adding new records and lines of code more easy and makes the file structure more consistent (like with comma example - all records have a comma after them). Linters usually have this rule warning the programmer about missing line in the end of file. GitHub also shows the warning in the pull request.

Reply:
Wow :) Yan, thanks! That's a  new insight for me :)

>  Linters usually have this rule warning the programmer about missing line in the end of file. GitHub also shows the warning in the pull request.

Reply:
Done )